sklearn/preprocessing/_data.py

```diff
@@ -1651,6 +1651,17 @@ def _more_tags(self):
         return {"allow_nan": True}
 
 
+@validate_params(
+    {
+        "X": ["array-like", "sparse matrix"],
+        "axis": [Options(Integral, {0, 1})],
+        "with_centering": ["boolean"],
+        "with_scaling": ["boolean"],
+        "quantile_range": [tuple],
+        "copy": ["boolean"],
+        "unit_variance": ["boolean"],
+    }
+)
```

Comment:
```suggestion
@validate_params(
    {
        "X": ["array-like", "sparse matrix"],
        "axis": [Options(Integral, {0, 1})],
    }
)
```

Since this function calls `RobustScaler`, which has been validated. I think we can avoid duplicate validation by just validating the remaining parameters.

Let's wait for a maintainer to decide.